.github/workflows/GHA-Scala-Instrumentation-Tests.yaml

```diff
@@ -118,7 +134,13 @@ jobs:
           LANGUAGE: scala
           SCALA_VERSION: scala${{ matrix.scala }}
           TEST_TYPE: instrumentation
-        run: ./gradlew --console=plain --parallel :instrumentation:test -Ptest${{ matrix.java-version }} -PnoInstrumentation -PnoScala -Pscala-${{ matrix.scala }} --continue --continue --scan -Dscan.uploadInBackground=false --build-cache
+        run: ./gradlew --console=plain --parallel :instrumentation:test -Ptest${{ matrix.java-version }} -PnoInstrumentation -PnoScala -Pscala-${{ matrix.scala }} --continue --scan -Dscan.uploadInBackground=false --build-cache
+
+      - name: Run instrumentation tests for Java ${{ matrix.java-version }} and scala ${{ matrix.scala }}
```

Comment:
This one is not specifying it does not have GE

Reply:
ok, thanks...hard to get these all "right"